docs/configuration.md

```diff
@@ -94,3 +94,10 @@ An identifier for the assumed role session
 The relative path that is used to fetch credentials inside and ECS instance.
 See [IAM Roles for Tasks](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-iam-roles.html) for more information.
 
+### s3PathStyleEndpoint
```

Comment:
`pathStyleEndpoint` seems to make more sense since....we are already in the context of an `S3Client`

Reply:
hmm, i see this is `global` config. but i stand by my point that this looks super strange:
```php
return new S3Client(['s3PathStyleEndpoint' => true]);
```

Reply:
Good point... Other clients use `PathStyleEndpoint`.
updated

Reply:
Fair. 
I was considering the scenario where we want to introduce an option for path style endpoint for Glacier? If someone wanted to use path style for Glacier but not S3 they couldn't do that.. 

But Im am wrong. Or. Im wrong most of the time. One could: 

```php
$glacier = new GlacierClient(['pathStyleEndpoint' => true]);
$s3 = new S3Client(['pathStyleEndpoint' => true]);
```

However, One cannot use the factory...

```php
$factory = new AwsFactory(['pathStyleEndpoint' => true]);
```

But I dont really care that much about the factory =). Using `pathStyleEndpoint` is good.

Reply:
I ended to the same conclusion. the same parameter can be used by several client ,it'll still work.

Regarding the factory, we have the same issue with all options: If someone want to use a region for S3 and another region for Glacier.

Reply:
Yeah. You are correct